webpack.common.js

```diff
@@ -120,6 +118,7 @@ module.exports = (publicPath = "auto") => {
         logEndpoint: JSON.stringify(logEndpoint),
         sentryDSN: JSON.stringify(process.env.SENTRY_DSN),
         sentryScriptUrl: JSON.stringify(process.env.SENTRY_SCRIPT_URL),
+        enableLogging: enableLogging,
```

Comment:
Define plugin will convert this boolean to string internally. But it is always better to explicitly stringify the value that you are defining as plugin. String values are often used as code fragments directly. The first answer to the stackoverflow questions helps understand this better.
https://stackoverflow.com/questions/39564802/why-does-webpacks-defineplugin-require-us-to-wrap-everything-in-json-stringify